pkgs/build-support/test-utilities.nix

```diff
@@ -0,0 +1,8 @@
+{ runCommand }:
+
+{
+    testVersion = command: args: version: runCommand "test-version" { meta.timeout = 60; } ''
```

Comment:
What about something like

```
{ package ? throw "testVersion: specify at least { package } or { command, version }",
  command ? "${packages}/bin/${package.pname or package.name}",
  args ? "--version",
  version ? package.version,
}:
```

Reply:
Indentation should be in two-space increments.

Maybe your editor has a plugin for `.editorconfig`?

Reply:
> Indentation should be in two-space increments.
> 
> Maybe your editor has a plugin for `.editorconfig`?

(oops, yeah I was trying out a different editor than I usually use. Curious that the editorconfig github hook didn't pick this up, btw.

Reply:
> What about something like

Good idea, iterated on it a bit further.